float64[3] single_xyz
bool is_left
---
bool success